experiment/schedule_measure_workers.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for starting instances to run measure workers."""
+import collections
+import posixpath
+
+from common import gcloud
+from common import gce
+from common import logs
+from common import queue_utils
+
+logger = logs.Logger('scheduler')  # pylint: disable=invalid-name
+
+
+def get_instance_group_name(experiment: str):
+    """Returns the name of the instance group of measure workers for
+    |experiment|."""
+    return experiment + '-measure-worker'
+
+
+def get_measure_worker_instance_template_name(experiment: str):
+    """Returns an instance template name for measurer workers running in
+    |experiment|."""
+    return experiment + '-measure-worker'
```

Comment:
this seems to create same name as get_instance_group_name ? something to differentiate.

Reply:
I think it's fine if they are named the same because they are for different things. I could add "template" to the template name and "group" to the group name but I'm not sure what we gain by doing this. And I don't think merging them is useful either. I like it as is.